### ISSUE TITLE
Confusion caused by "speech" extension category and "sounds" category

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-gui#586]
**TL;DR:** During the Extension Categories External Playtest (8/3/17), we noticed users getting confused by the difference between the “speech” and “sounds” categories, when using the Speech extension.

The users weren't sure where to find which blocks, or when to expect which block to affect which kind of audio output. 

For example, if you record a sound of yourself speaking, you should use the Sounds > Play sound / Play sound until done blocks to play the sound, but if you make the computer say out loud the same words, you should use the Speech > Speak block. 

To further complicate matters, once we add the say & think blocks, those will be located under Looks. (And are another way to make the computer "say" something.)

Related to https://github.com/LLK/scratch-gui/issues/531.

**Comment by chrisgarrity** [comment]
Moved to LLK/observations#1
